utils/brand.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# shellcheck source=utils/lib.sh
+. /dev/null
```

Comment:
why is it needed?

Reply:
Its needed to prevent an warning [SC2154](https://www.shellcheck.net/wiki/SC2154) "var is referenced but not assigned" from shellcheck ..

Usually you wrote

```bash
# shellcheck source=utils/lib.sh
. utils/lib.sh
```

This will source `utils/lib.sh` and gives shellcheck the hint, that variable names in *this* (`utils/brand.sh`) script are defined in the sourced file.

Since `utils/brand.sh` is sourced by the main script `utils/searxng.sh` which already sources `utils/lib.sh` we don't want to source it again .. but we need to give shellcheck a hint from where environment variables used here are coming from.

--> Thats why we source from `/dev/null` instead:

```
# shellcheck source=utils/lib.sh
. /dev/null
```

There is an ticket in the shellcheck project about:

- https://github.com/koalaman/shellcheck/issues/356#issuecomment-853515285

Reply:
Ok I understand this is related to `REPO_URL`.

Reply:
> Ok I understand this is related to `REPO_URL`.

You mean `REPO_ROOT` I guess .. then the answer is: *yes*